src/views/components/database/dataBlocks/DataFieldgroupField.tsx

```diff
@@ -30,8 +34,8 @@ const FieldData = styled.span`
   user-select: text;
 `;
 
-export const DataFieldgroupField = ({ label, data }: DataFieldgroupFieldProps) => (
-  <Fieldgroup>
+export const DataFieldgroupField = ({ label, data, width }: DataFieldgroupFieldProps) => (
```

Comment:
```suggestion
export const DataFieldGroupField = ({ label, data, width }: DataFieldGroupFieldProps) => (
```

Petit oublie de majuscule ici

Reply:
Ca fait 3 ans que c'est comme ça ^^' En plus pour homogénéiser il faudrait également renommer DataFieldgroup et DataFieldsetField mais ça fait trop de changements pour pas grand chose. Et enfin renommer les fichiers mais puisque Windows n'est pas case sensitive ça va buguer et ça va faire comme les fichiers "index" et "Index" qui ont bug pendant 2 ans et demi.